app/avatar/migrations/0002_auto_20190114_1929.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 2.1.4 on 2019-01-14 19:29
+
+import avatar.utils
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+import economy.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0007_remove_profile_avatar'),
+        ('avatar', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BaseAvatar',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_on', models.DateTimeField(db_index=True, default=economy.models.get_time)),
+                ('modified_on', models.DateTimeField(default=economy.models.get_time)),
+                ('active', models.BooleanField(default=False)),
+                ('svg', models.FileField(blank=True, help_text='The avatar SVG.', null=True, upload_to=avatar.utils.get_upload_filename)),
+                ('png', models.ImageField(blank=True, help_text='The avatar PNG.', null=True, upload_to=avatar.utils.get_upload_filename)),
+                ('hash', models.CharField(max_length=16)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.DeleteModel(
+            name='Avatar',
+        ),
+        migrations.CreateModel(
+            name='CustomAvatar',
+            fields=[
+                ('baseavatar_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='avatar.BaseAvatar')),
+                ('recommended_by_staff', models.BooleanField(default=False)),
+                ('config', django.contrib.postgres.fields.jsonb.JSONField(default=dict, help_text='The JSON configuration.')),
+            ],
+            options={
+                'abstract': False,
+            },
+            bases=('avatar.baseavatar',),
+        ),
+        migrations.CreateModel(
+            name='SocialAvatar',
```

Comment:
this table is empty, I wonder if multitable inheritance is good idea in this case, maybe proxy models could be better